configs/default.json

```diff
@@ -14,4 +11,4 @@
             "standard": 0
```

Comment:
What about variation groups?
How can we ensure that variation groups will be not removed from catalog?

Reply:
Variation groups will not be removed if the corresponding master product will not be removed too, works as expected. I'll add product with variation groups to test data.